config/sync/language/da/core.date_format.short.yml

```diff
@@ -1 +1,2 @@
 label: 'Standard kort dato'
+pattern: 'm/d/Y - H:i'
```

Comment:
Is this supposed to be here or a mistake? 

This is a change i keep seeing repeated in config exports.

Reply:
I dunno why it happens - either way, I've just manually removed it now.